src/logger/POSTLogger.ts

```diff
@@ -53,6 +62,14 @@ export default class MeetingSessionPOSTLogger implements Destroyable {
     window.removeEventListener('unload', this.eventListener);
   }
 
+  get metadata(): Record<string, string> | undefined {
```

Comment:
Do we need to do this or can we just set metadata to be a public prop?

Reply:
I kept it as getter setter so that if in future we want to validate on size or properties, a set method would be a good way to control it. In case of public, we would have no such control. Hence, I think it would be good to use setter and getter here.

Reply:
Don't think change it to get/set from public is a breaking change right? I prefer to do it whenever we need to do it :)

Reply:
Changed metadata to a public property now.